rmgpy/data/kinetics/family.py

```diff
@@ -3350,7 +3350,8 @@ def prune_tree(self, rxns, thermo_database=None, max_rxns_to_reopt_node=100, fix
             if parent and len(template_rxn_map[parent.label]) < max_rxns_to_reopt_node:
                 parent.children.remove(entry)
                 del self.groups.entries[key]
-                parent.item.clear_reg_dims()
+            else:
+                entry.item.clear_reg_dims()
```

Comment:
So when we cascade we add new reactions. Because there are new reactions at each node some of the modifications of groups that used to not affect what reactions matched the group (because they matched everything or nothing) may affect what reactions are matched. This means the old regularization dimensions are no longer necessarily correct and need recalculated so we clear them.

Reply:
I see. That makes sense.